commands/fly.go

```diff
@@ -26,7 +26,7 @@ type FlyCommand struct {
 	Watch   WatchCommand   `command:"watch"   alias:"w" description:"Stream a build's output"`
 
 	Containers ContainersCommand `command:"containers" alias:"cs" description:"Print the active containers"`
-	Hijack     HijackCommand     `command:"hijack"     alias:"intercept" alias:"i" description:"Execute a command in a container"`
+	Intercept  InterceptCommand  `command:"intercept"     alias:"intercept" alias:"i" description:"Execute a command in a container"`
```

Comment:
Either way having an alias that's the same command as the official "command", as seen here

```
`command:"intercept"     alias:"intercept"...
```

is most definitely redundant.